index.html

```diff
@@ -285,107 +271,194 @@ <h2>
-        implementation.
-      </p>
-      <section id="multiple-batteries">
-        <h2>
-          Multiple batteries
-        </h2>
+      <aside class="note">
+        If the user agent is <a>unable to report the battery status
+        information</a>, the {{BatteryManager}}'s internal slots will remain
+        with their default values, which emulate a fully charged and plugged in
+        battery. This is aimed at reducing the potential for fingerprinting and
+        preventing applications from degrading performance, if the battery
+        status information is not made available, for example.
+      </aside>
+      <section>
+        <h3>
+          Internal slots
+        </h3>
         <p>
-          If a hosting device contains more than one battery,
-          <a>BatteryManager</a> SHOULD expose an unified view of the batteries.
+          {{BatteryManager}} instances are created with the following
+          <a data-cite=
+          "ECMASCRIPT#sec-object-internal-methods-and-internal-slots">internal
+          slots</a>:
         </p>
+        <table class="simple">
+          <thead>
+            <tr>
+              <th>
+                Internal slot
+              </th>
+              <th>
+                Initial value
+              </th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td>
+                <dfn>[[\Charging]]</dfn>
+              </td>
+              <td>
+                `true`
+              </td>
+            </tr>
+            <tr>
+              <td>
+                <dfn>[[\ChargingTime]]</dfn>
+              </td>
+              <td>
+                0
+              </td>
+            </tr>
+            <tr>
+              <td>
+                <dfn>[[\DischargingTime]]</dfn>
+              </td>
+              <td>
+                Positive Infinity
+              </td>
+            </tr>
+            <tr>
+              <td>
+                <dfn>[[\Level]]</dfn>
+              </td>
+              <td>
+                1.0
+              </td>
+            </tr>
+          </tbody>
+        </table>
+        <section>
+          <h4>
+            The `[[Charging]]` internal slot
+          </h4>
+          <p>
+            The {{BatteryManager/[[Charging]]}} internal slot represents the
+            charging state of the system's battery. It MUST be set to `false`
+            if the battery is discharging, and set to `true` if the battery is
+            charging, the implementation is unable to report the state, or
+            there is no battery attached to the system, or otherwise.
+          </p>
+          <p>
+            When the system battery's charging state changes, the user agent
+            must run the <a>update the battery status and notify</a> algorithm
+            with {{BatteryManager/[[Charging]]}}, `true` or `false` depending
+            on whether the battery is charging or discharging, and
+            "`chargingchange`".
```

Comment:
Link to the definition of `"chargingchange"` here, and the same for the other events below.

Reply:
Done. I was following the WHATWG style from e.g. https://notifications.spec.whatwg.org/#object-members that does not define the items in the second column of the event handler table. I've now switched to `<dfn>`ing both columns, PTAL.